README.md

```diff
@@ -2,30 +2,27 @@
 
 <img align="right" src="doc/manual/figures/ale.gif" width=50>
 
-# Arcade-Learning-Environment: An Evaluation Platform for General Agents
+# The Arcade Learning Environment
 
-The Arcade Learning Environment (ALE) -- a platform for AI research.
+The Arcade Learning Environment (ALE) is a simple object-oriented framework that allows researchers and hobbyists to develop AI agents for Atari 2600 games. It is built on top of the Atari 2600 emulator [Stella](https://stella-emu.github.io/) and separates the details of emulation from agent design. This [video](https://www.youtube.com/watch?v=nzUiEkasXZI) depicts over 50 games currently supported in ALE.
```

Comment:
I like "the ALE" better than "ALE". We're using both here -- can you switch to "the ALE" only?